Objects/longobject.c

```diff
@@ -1209,7 +1209,7 @@ _PyLong_AsByteArray(PyLongObject* v,
         *p = (unsigned char)(accum & 0xff);
         p += pincr;
     }
-    else if (j == n && n > 0 && is_signed) {
+    else if (j == n && is_signed) {
```

Comment:
This change introduced an undefined behavior: `unsigned char msb = *(p - pincr);` read is out of the bounds if n == 0. I wrote https://github.com/python/cpython/pull/138873 to fix it.

Reply:
Oh, sorry.  Thanks!